bkg/v2/BKG_v2.0.0.yaml

```diff
@@ -6959,22 +6959,30 @@ components:
           description: |
             Number of requested equipment units.
           example: 3
-        emptyContainerPositionings:
+        containerPositionings:
           type: array
           description: |
-            Possibility to add multiple `ECP` (Empty Container Positioning) locations. If multiple `ECP` locations are provided, all locations must be within the same city.
+            A list of date and time and locations of the customer facility where the container(s) will be loaded. If multiple Container positioning locations (`CPO`) are provided (multi-stop), the first location is where the empty container will be stuffed first. The order in which the `CPO` locations should be visited is implicitly defined by the shipper based on the date and time provided per location.
             
-            **Conditional:** Only applicable to carrier haulage service at origin (`Receipt type at origin = SD`).
+            **Conditional:** Only applicable to carrier haulage service at origin (`Receipt type at origin = 'SD'`). 
           items:
-            $ref: '#/components/schemas/EmptyContainerPositioning'
-        emptyContainerPickupDateTime:
-          type: string
-          format: date-time
+            $ref: '#/components/schemas/ContainerPositioning'
+        emptyContainerPickup:
+          type: object
+          title: Empty Container Pickup
           description: |
-            The date and time requested by the shipper for the pick-up of the empty container(s) at the `Empty Container Depot Release Location` (DRL), if provided.
-          example: '2024-07-05T09:03:00-02:00'
-        emptyContainerDepotReleaseLocation:
-          $ref: '#/components/schemas/EmptyContainerDepotReleaseLocation'
+            The date and time and location for the empty container pick-up.
+            
+            **Condition:** Only applicable to merchange haulage service at origin (`Receipt type at origin = 'CY'`).
+          properties:
+            dateTime:
+              type: string
+              format: date-time
+              description: |
+                The date and time requested by the shipper for the pick-up of the empty container(s) at the `Empty Container Depot Release Location` (DRL), if provided.
+              example: '2024-07-05T09:03:00-02:00'
+            location:
```

Comment:
Should this be depotReleaseLocation or location?

Reply:
Text is synced with issue now